Deutsches Arzteblatt.js

```diff
@@ -0,0 +1,255 @@
+}
+
+/*
+	***** BEGIN LICENSE BLOCK *****
+
+	Copyright © 2018 Sebastian Fleer
+
+	This file is part of Zotero.
+
+	Zotero is free software: you can redistribute it and/or modify
+	it under the terms of the GNU Affero General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	Zotero is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+	GNU Affero General Public License for more details.
+
+	You should have received a copy of the GNU Affero General Public License
+	along with Zotero. If not, see <http://www.gnu.org/licenses/>.
+
+	***** END LICENSE BLOCK *****
+*/
+
+function detectWeb(doc, url) {
+	if (url.match(/archiv\/\d+/) || url.match(/aid\=\d+/)) {
+		return "journalArticle";
+	}
+}
+
+function doWeb(doc, url) {
+	var type = detectWeb(doc, url);
+	if (type == "journalArticle") {
+		scrape(doc, url);
+	}
+}
+
+function scrape(doc, url) {
+	var newItem = new Zotero.Item(detectWeb(doc, url));
+	
+	if (newItem.itemType == "journalArticle") {
+		// detect article ID
+		var tmp = "null";
+		if (url.match(/archiv\/\d+/)) { // normal article page
+			tmp = url.match(/archiv\/\d+/);
+		} else if (url.match(/aid\=\d+/)) { // article page in search results
+				tmp = url.match(/aid\=\d+/);
+		}
+		var articleID = String(tmp).match(/\d+/);
+
+		// use BibTeX Translator
+		var translator = Zotero.loadTranslator("import");
+		translator.setTranslator("9cb70025-a888-4a29-a210-93ec52da40d4");
+	
+		translator.setHandler("itemDone", function(obj, item) {
+			item.attachments.push({
+				title:"Snapshot",
+				document:doc
+			});
+	
+			// PDFs are stored under https://www.aerzteblatt.de/pdf/<volume>/<issue>/pdfPageNum.pdf
+			// pdfPageNum is in the form a<firstpage> or m<firstpage>, depending on the journal part
+			// Don't fetch PDF by now
+			
+			item.complete();
+		});
+	
+		// Fetch BibTex file for article ID and set encoding to ISO-8859-15
+		var bibtexURL = "https://www.aerzteblatt.de/callback/citemgr.asp?fmt=bibtex&id=" + articleID;
+		Zotero.Utilities.HTTP.doGet(bibtexURL, function(text) {
+			translator.setString(text);
+			translator.translate();
+		}, null, "ISO-8859-15");
+	}
+}
+/** BEGIN TEST CASES **/
+var testCases = [
+	{
+		"type": "web",
+		"url": "https://www.aerzteblatt.de/archiv/202039/Oekonomisierung-der-Medizin-Zur-Pathogenese-der-Oekonomisierung",
+		"items": [
+			{
+				"itemType": "journalArticle",
+				"title": "Ökonomisierung der Medizin: Zur Pathogenese der Ökonomisierung",
+				"creators": [
+					{
+						"firstName": "Christian",
+						"lastName": "Thielscher",
+						"creatorType": "author"
+					}
+				],
+				"date": "2018",
+				"issue": "43",
+				"itemID": "DAE202039",
+				"libraryCatalog": "Deutsches Ärzteblatt",
+				"pages": "A-1946-",
+				"publicationTitle": "Dtsch Arztebl International",
```

Comment:
Maybe this belongs in `journalAbbreviation` (although I believe `International` should be `int.`), but not `publicationTitle`, where it should be spelled out completely.